templates/common/root/_Gruntfile.js

```diff
@@ -280,6 +284,33 @@ module.exports = function (grunt) {
           sourcemap: true
         }
       }
+    },<% } else { %>
```

Comment:
this shouldn't be an else. it should see if sass is yes otherwise you'll have sass in the grunt for no reason